tests/TestCase/Policy/OrmResolverTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         1.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authorization\Test\TestCase\Policy;
+
+use Authorization\Policy\Exception\MissingPolicyException;
+use Authorization\Policy\OrmResolver;
+use Cake\ORM\Entity;
+use Cake\TestSuite\TestCase;
+use InvalidArgumentException;
+use TestApp\Model\Entity\Article;
+use TestApp\Model\Table\ArticlesTable;
+use TestApp\Policy\ArticlePolicy;
+use TestApp\Policy\ArticlesTablePolicy;
+use TestApp\Policy\TestPlugin\BookmarkPolicy;
+use TestPlugin\Model\Entity\Bookmark;
+use TestPlugin\Model\Entity\Tag;
+use TestPlugin\Policy\TagPolicy;
+
+class OrmResolverTest extends TestCase
+{
+    public function testGetPolicyUnknownObject()
+    {
+        $this->setExpectedException(InvalidArgumentException::class);
```

Comment:
This method is deprecated and missing in PHPUnit 6.0, use `expectException()` instead.

Reply:
Fixed now.